src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -253,6 +261,31 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
     this.scrollToQuestion();
   }
 
+  private loadCourseInfo(): void {
+    this.isCourseLoading = true;
+    let request: Observable<Course>;
+    switch (this.intent) {
+      case Intent.STUDENT_SUBMISSION:
+        if (this.moderatedPerson || this.previewAsPerson) {
+          request = this.courseService.getCourseAsInstructor(this.courseId);
+        } else {
+          request = this.courseService.getCourseAsStudent(this.courseId, this.regKey);
+        }
+        break;
+      case Intent.INSTRUCTOR_SUBMISSION:
+        request = this.courseService.getCourseAsInstructor(this.courseId, this.regKey);
+        break;
+      default:
+        this.isCourseLoading = false;
+        return;
+    }
+    request.subscribe((resp: Course) => {
+      this.courseName = resp.courseName;
+      this.courseInstitute = resp.institute;
+      this.isCourseLoading = false;
+    });
```

Comment:
Do we need to handle the case where the request throws an error? Or is this out of scope for this PR.

Current behavior is that it doesn't stop the loading spinner.
<img width="910" alt="Screen Shot 2022-03-31 at 12 27 40 AM" src="https://user-images.githubusercontent.com/60355570/160884760-0885bf05-71e0-43b0-9995-95baf946ec8f.png">